libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -965,20 +966,21 @@ class MapboxNavigation(
      * @param feedbackType one of [FeedbackEvent.Type]
      * @param description description message
      * @param feedbackSource one of [FeedbackEvent.Source]
-     * @param screenshot encoded screenshot (optional)
+     * @param screenshot encoded screenshot
      * @param feedbackSubType array of [FeedbackEvent.SubType] (optional)
      * @param feedbackMetadata use it to attach feedback to a specific passed location.
      * See [FeedbackMetadata] and [FeedbackMetadataWrapper]
      *
      * @see [FeedbackHelper.getFeedbackSubTypes]
      * to retrieve possible feedback subtypes for a given [feedbackType]
+     * @see [ViewUtils] to capture and encode screenshots
      */
     @ExperimentalPreviewMapboxNavigationAPI
     fun postUserFeedback(
```

Comment:
As we're breaking SemVer was thinking if we could compact the overloaded `postUserFeedback` APIs into one with `feedbackMetadata` as `null` by default.